provider/lib/cronAlarm.js

```diff
@@ -104,7 +104,7 @@ module.exports = function(logger, newTrigger) {
         var method = "distributeCronAlarm";
 
         var cronFields = (trigger.cron + '').trim().split(/\s+/);
-        if (trigger.strict !== 'true' && cronFields.length === 5 && delayLimit !== 0) {
+        if (!trigger.strict && cronFields.length === 5 && delayLimit !== 0) {
```

Comment:
If a `strict` parameter does not exist, it will add some delays by default.
So an OW operator can choose whether to enable this feature by setting up `ALARM_DELAY_LIMIT`.

I think the documentation should be updated as well.